wallet/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1593,6 +1597,54 @@ instance BuildableSafeGen WalletSoftwareUpdate where
         updBlockchainVersion
         updScriptVersion
 
+data WalletBalance = WalletBalance
+  { wbBalance  :: V1 (Core.Coin)
+  , wbWalletId :: Text
+  } deriving (Show, Eq, Generic)
+deriveJSON Aeson.defaultOptions ''WalletBalance
+
+instance ToSchema WalletBalance where
+    declareNamedSchema =
+        genericSchemaDroppingPrefix "wb" (\(--^) props -> props
+            & "balance"
+            --^ "the balance of a given public key"
```

Comment:
`walletId` is missing here. Although, I have doubt about its value in the response? Wouldn't make more sense to have it as a path parameter on the endpoint?

Reply:
id rather avoid re-writting all of that crypto code in JS
this endpoint is handling the heavy lifting of converting a public key into a walletid, so JS doesnt need to know how